tap_github/organization_streams.py

```diff
@@ -0,0 +1,168 @@
+"""User Stream types classes for tap-github."""
+
+from typing import Dict, List, Optional, Iterable, Any
+
+from singer_sdk import typing as th  # JSON Schema typing helpers
+
+from tap_github.client import GitHubRestStream
+
+
+class OrganizationStream(GitHubRestStream):
+    name = "organizations"
+
+    @property
+    def path(self) -> str:  # type: ignore
```

Comment:
Is there a reason this one is a property as opposed to a simple string like in other streams?

Reply:
good catch, I just copied another stream haha